lib/screens/lobby/list_of_created_games.dart

```diff
@@ -1,15 +1,33 @@
 import 'package:flutter/material.dart';
```

Comment:
after renaming Lobby folder to lobby, this pr has changes for "lobby/list_of_created_games.dart" & "Lobby/list_of_created_games.dart". idk why, and i tried to play around with it but it would still stage both files (first doesnt exist) when i would commit code. 
im really hoping it just goes away when we merge it dev